util/py/scripts/mapfile_to_json.py

```diff
@@ -425,9 +428,35 @@ def sum_children(self, item):
         return csz
 
 
+def generate_map(target):
+    # Define the Bazel command as a list of arguments
+    # bazel_cmd = ["bazel", "build", "//sw/device/tests/crypto:otcrypto_export_size"]
+    map_file = None
+    bazel_cmd = ["bazel", "build", target]
+    try:
+        result = subprocess.run(bazel_cmd, check=True, capture_output=True, text=True)
+        print(result)
+        # Search output for the .map file line
+        for line in result.stderr.splitlines():
+            print(line)
+            if line.strip().endswith('.map'):
+                # The file path is the whole line or first token
+                map_file = line.strip().split()[0]
+        print("Generate map file from target succeeded:\n", result.stdout)
+    except subprocess.CalledProcessError as e:
+        print("Generate map file from target failed:\n", e.stderr)
+
+    return map_file
+
+
```

Comment:
I'm ambivalent about this, but: it is possible for an `opentitan_binary` to target to generate multiple mapfiles depending on the number `exec_envs` passed in.  If we want to keep this, there should be some strategy for selecting a particular mapfile.

Reply:
Yes, we can file an issue and handle it in a followup.